src/modules/hero/hero.component.scss

```diff
@@ -11,21 +10,20 @@
 }
 
 .stache-hero-body {
-  margin-top: 80px;
-  margin-bottom: 100px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 100px 0 80px;
+  max-width: 75%;
   color: #fff;
   text-align: center;
-  max-width: 550px;
-  position: relative;
   z-index: 1;
 }
 
 .stache-hero-overlay {
   background-color: #000;
-  opacity: .35;
+  opacity: 0.4;
   position: absolute;
-  top: 0;
-  right: 0;
-  bottom: 0;
-  left: 0;
+  height: 100%;
```

Comment:
You can add height and width, here, but the positional properties (top, left, etc.) are also important for cross-browser support.